src/egon/data/datasets/plotdatascn.py

```diff
@@ -0,0 +1,191 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# File description
+"""
+Created on Tue May 24 14:42:05 2022
+Plotdatascn.py defines functions to plot to provide a better context of the different parameters part of
+scenarios eGon2035 and eGon100RE .
+@author: Alonso
+
+""" 
+import logging
+import os
+from matplotlib import pyplot as plt
+import matplotlib as mpl
+import pandas as pd
+from math import sqrt, log10
+from pyproj import Proj, transform
+from egon.data import db
+import egon.data.config
+import geopandas as gpd
+import tilemapbase
+from math import sqrt, log10
+from pyproj import Proj, transform
+from matplotlib_scalebar.scalebar import ScaleBar
```

Comment:
matplotlib-scalebar is also not part of the libraries installed with egon-data. Please include a note about it too.

Reply:
done